Project.toml

```diff
@@ -5,12 +5,20 @@ version = "2.0.9"
 [deps]
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 CommonSolve = "38540f10-b2f7-11e9-35d8-d573e4eb0ff2"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
```

Comment:
is ForwardDiff supposed to be here? i know that in future versions, packages in `weakdeps` will be ignored from `deps`, but i don't think that is the case for julia <1.9

Reply:
If it's added here, ForwardDiff will be a regular dependency on Julia < 1.9 and (due to the lines below) a weak dependency on >= 1.9.

Reply:
Thanks both of you, I've removed the line.